cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/KinesisConsumerIT.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package com.yugabyte.cdcsdk.testing;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.UnsupportedEncodingException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.BindMode;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.BasicSessionCredentials;
+import com.amazonaws.services.kinesis.AmazonKinesisClient;
+import com.amazonaws.services.kinesis.model.GetRecordsRequest;
+import com.amazonaws.services.kinesis.model.GetRecordsResult;
+import com.amazonaws.services.kinesis.model.GetShardIteratorRequest;
+import com.amazonaws.services.kinesis.model.Shard;
+import com.amazonaws.services.kinesis.model.ShardIteratorType;
+import com.yugabyte.cdcsdk.testing.util.CdcsdkTestBase;
+import com.yugabyte.cdcsdk.testing.util.UtilStrings;
+
+/**
+ * Release test that verifies inserts, updates and deletes on a YugabyteDB
+ * database and
+ * writing to Amazon Kinesis
+ *
+ * @author Sumukh Phalgaonkar
+ */
+
+public class KinesisConsumerIT extends CdcsdkTestBase {
+
+    private static String stream_name = "dbserver1.public.test_table";
+
+    @BeforeAll
+    public static void beforeClass() throws Exception {
+        initializeContainers();
+
+        initHelpers(true, false, false);
+    }
+
+    @BeforeEach
+    public void beforeEachTest() throws Exception {
+        ybHelper.execute(UtilStrings.getCreateTableYBStmt(DEFAULT_TABLE_NAME));
+
+        cdcsdkContainer = TestHelper.getCdcsdkContainerForKinesisSink(ybHelper, "public." + DEFAULT_TABLE_NAME)
+                .withCreateContainerCmdModifier(cmd -> cmd.withUser(System.getenv("USERID")));
+        cdcsdkContainer.withNetwork(containerNetwork);
+
+        cdcsdkContainer.withFileSystemBind(System.getenv("AWS_SHARED_CREDENTIALS_FILE"),
```

Comment:
Is this required since you are also setting Secret & Access key from env variables below?

Reply:
Yes. This is for setting the profile for cdcsdk container. Whereas the env variables are used for the Kinesis client.